start-testnet-reth-bootnode.sh

```
@@ -0,0 +1,16 @@
+# !/bin/sh
```

Shellcheck (0.10.0)

[error] Remove spaces between # and ! in the shebang.

(SC1115)